core/domain/state_domain.py

```diff
@@ -2722,6 +2726,15 @@ def update_next_content_id_index(self, next_content_id_index):
         """
         self.next_content_id_index = next_content_id_index
 
+    def update_linked_skill_id(self, linked_skill_id):
+        """Update the state linked skill id attribute.
+
+        Args:
+            linked_skill_id: str. The linked skill id to state.
+        """
+
```

Comment:
You don't need the extra newline here.  Follow the existing coding patterns.

Reply:
Done